src/components/Voting.jsx

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from "react";
+import { incrementVotes} from "../api";
+const Voting = ({votes, article_id}) => {
+    const [upVote, setUpvote] = useState("upvote");
+  const [downVote, setDownvote] = useState("downvote");
+  const [currVote, setCurrVote] = useState(votes);
+  const [upVoteClicked, setUpClicked] = useState();
+  const [downVoteClicked, setDownClicked] = useState();
+
+
+  useEffect(() => {
+}, [currVote])
+const handleUpvote = () => {
+  if (!upVoteClicked) {
+    setDownClicked(false);
+    setDownvote("downvote");
+    
+    setUpvote("upvoted");
+    setCurrVote(currVote + 1);
+  
+    setUpClicked(true);
+
+    incrementVotes(article_id, 1).catch(err => {
+        console.log(err)
+        setCurrVote(currVote-1)
+    })
+  }
+};
+
+const handleDownvote = () => {
+  if (!downVoteClicked) {
+    setUpClicked(false);
+    setUpvote("upvote");
+
+    setDownvote("downvoted");
+    setCurrVote(currVote - 1);
+  
+    setDownClicked(true);
+    incrementVotes(article_id, -1).catch(err => {
+        console.log(err)
+        setCurrVote(currVote+1)
+    })
+  }
+};
+
+    return (
+        <aside className="votes">
+        <span
+          class={upVote}
+          onClick={() => {
+            handleUpvote();
+          }}
```

Comment:
You can just pass the handleUpvote to onClick because you don't want to pass it anything specific e.g.

```suggestion
          onClick={handleUpvote}
```